example_data/config.yaml

```diff
@@ -22,8 +22,9 @@ speakers:
   - id: 1
     name: Laura
     description: The main host. She explains topics clearly using anecdotes and analogies, teaching in an engaging and captivating way.
-    voice_profile: Laura's voice is exciting and fast in delivery with very clear audio and no background noise.
+    voice_profile: female_1
   - id: 2
     name: Jon
     description: The co-host. He keeps the conversation on track, asks curious follow-up questions, and reacts with excitement or confusion, often using interjections like hmm or umm.
-    voice_profile: Jon's voice is calm with very clear audio and no background noise.
+    voice_profile: male_1
+device: "cpu"
```

Comment:
I think it doesn't make sense to have this as a parameter. 
We haven't tested gpu support yet and once we have it, I think is better to use `torch.cuda.is_available` to set the device automatically.

```suggestion
```